app/src/main/java/com/e/salesapp/ProductPurchasedListActivity.java

```diff
@@ -0,0 +1,47 @@
+package com.e.salesapp;
+
+import android.os.Bundle;
+import android.os.PersistableBundle;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.e.salesapp.adapters.ProductAdapter;
+import com.e.salesapp.adapters.ProductPurchasedAdapter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ProductPurchasedListActivity extends AppCompatActivity {
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_products_purchased_list);
+        Toolbar toolbar = (Toolbar) findViewById(R.id.cartListToolbar);
+        setSupportActionBar(toolbar);
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+        getSupportActionBar().setDisplayShowHomeEnabled(true);
+
+        RecyclerView rvMovies = findViewById(R.id.rv_cartProducts);
+        List<String> cartList = new ArrayList<>();
+
+        //fake data
+        cartList.add("Fruits");
+        cartList.add("Vegetables");
+
+        ProductPurchasedAdapter mProductAdapter = new ProductPurchasedAdapter(cartList, this);
+
+        rvMovies.setHasFixedSize(true);
+        rvMovies.setAdapter(mProductAdapter);
+        rvMovies.setLayoutManager(new LinearLayoutManager(this));
```

Comment:
I think this piece of code should goes outside, it will be better if you create a new method and define inside it this piece of code